webui/react/Makefile

```diff
@@ -64,15 +64,20 @@ check-prettier: check-prettier-js check-prettier-css check-prettier-misc
 check: check-eslint check-stylelint check-prettier check-package-lock
 
 
+.PHONY: fmt-js-eslint
+fmt-js-eslint:
+	npm run eslint -- --fix
+.PHONY: fmt-js-prettier
+fmt-js-prettier:
+	npm run format:js > /dev/null
 .PHONY: fmt-js
-fmt-js:
-	npm run format:js
+fmt-js: fmt-js-eslint fmt-js-prettier
```

Comment:
Did you see the difference between with `fmt-js-eslint` and without it? 
I believe that there's no difference but probably good to have both just in case

Reply:
I tried to demo this behavior with the commit sequence of the branch https://github.com/determined-ai/determined/pull/5096/commits
- checkout the first commit and run `check` see it fails
- run fmt then run check and it still fails
- checkout the next commit. fmt. and check passes now

Reply:
you're right. we should def have `fmt-js-eslint` then. Thanks for the clarification